### PR TITLE
add taggle v1.7.1 with npm auto-update config

### LINKED ISSUE #7925: [Request] Add Taggle
**Library name:** Taggle
**Git repository url:** https://github.com/okcoker/taggle.js
**npm package url(optional):** https://www.npmjs.com/package/taggle
**License(s):** MIT
**Official homepage:** https://sean.is/poppin/tags
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #7925 
@pvnr0082t please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/okcoker/taggle.js
 * Official website (optional, not the repository): https://sean.is/poppin/tags
 * NPM package url (optional): https://www.npmjs.com/package/taggle
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 221
   - Count of watchers: 12
   - Count of forks: 50
 * NPM download stats (optional):
   - Downloads in the last day: 22
   - Downloads in the last week: 335
   - Downloads in the last month: 1653

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7925, cc @okcoker

### CODE REVIEW

File: ajax/libs/taggle/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "tagging",
+    "standalone",
+    "micro",
+    "library"
+  ],
+  "license": "MIT",
+  "npmName": "taggle",
+  "npmFileMap": [
+    {
+      "basePath": "example/js",
```
Comment (pvnr0082t) [inline ajax/libs/taggle/package.json]: @kennynaoh It'll be better to not use `js` or `css` as base path. please leave `basepath` blank.

File: ajax/libs/taggle/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "files": [
+        "taggle*"
+      ]
+    },
+    {
+      "basePath": "src",
```
Comment (pvnr0082t) [inline ajax/libs/taggle/package.json]: @kennynaoh Why do you use `src` as basepath? Don't we need to keep structure? @PeterDaveHello Is it ok to use `src` as basepath?

Comment (PeterDaveHello) [inline ajax/libs/taggle/package.json]: It depends, looks fine here.

File: ajax/libs/taggle/package.json (hunk 1 above)
Comment (kennynaoh) [inline ajax/libs/taggle/package.json]: leave `basePath` blank or use `example` as basePath?

Comment (pvnr0082t) [inline ajax/libs/taggle/package.json]: I'll leave it blank. How do you think?

Comment (kennynaoh) [inline ajax/libs/taggle/package.json]: like this
```"basePath": "example",```
```"files": [```
 ```"js/taggle*" ```
```] ```
Is it correct?

Comment (pvnr0082t) [inline ajax/libs/taggle/package.json]: Maybe it is ok. I'm not sure.

```json
"basePath": "example",
"files": [
    "js/taggle*" 
] 
```

```json
"basePath": "",
"files": [
    "example/js/taggle*" 
] 
```
 Let's ask @PeterDaveHello which one is better.

**Comment by PeterDaveHello** [comment]
hmmmm ... using `example` as base path is also wired ...

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated the `npmFileMap`.
Please help me review it again.
Thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/taggle/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "micro",
+    "library"
+  ],
+  "license": "MIT",
+  "npmName": "taggle",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "example/js/taggle*"
```
Comment (PeterDaveHello) [inline ajax/libs/taggle/package.json]: Thanks will make the url contain `example/js`,  makes no sense.

Comment (kennynaoh) [inline ajax/libs/taggle/package.json]: @PeterDaveHello 
So how should I do?
Thanks!

Comment (PeterDaveHello) [inline ajax/libs/taggle/package.json]: Example is not a distribution purpose directory, it's wired to get files from an example, unless you've confirmed with author to make sure that the example will be updated in every release.

**Comment by kennynaoh** [comment]
@pvnr0082t , @PeterDaveHello 
I have confirmed with author by https://github.com/okcoker/taggle.js/issues/53
So which pattern should I use?

`"basePath": "example", `
`"files:[ `
`"js/taggle*" `
`] `

OR

`"basePath": "", `
`"files:[ `
`"examplejs/taggle*" `
`] `

Thanks!

**Comment by PeterDaveHello** [comment]
The upper one.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated the npmFileMap.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/taggle/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "taggle",
+  "filename": "js/taggle.min.js",
+  "version": "1.7.1",
+  "description": "Taggle is a dependency-less tagging library",
+  "author": "Sean Coker <sean@seancoker.com>",
+  "url": "http://sean.is/poppin/tags",
```
Comment (pvnr0082t) [inline ajax/libs/taggle/package.json]: Oh! @kennynaoh I think `url` can be removed.

**Comment by PeterDaveHello** [comment]
using js as base path is not a good idea, however, put the src and minified js in different level of directories still bad, let's use "example/js" as base path this time.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated the npmFileMap.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**